modules/admin/authentication.py

```diff
@@ -0,0 +1,21 @@
+from uuid import uuid4
+from .user import User
+
+from . import context
+
+USERNAME = "admin"
```

Comment:
I guess this because you don’t really want to deal with database and encryption, right?

Reply:
Correct. We also need to discuss the authentication and authorization processes.

Reply:
OK we can have a meeting later today or tomorrow.